scripts/azure-pipelines/owners-db/file_script.ts

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env node
+import * as fs from "fs";
+import * as path from "path";
+
+const keyword = "/include/";
```

Comment:
```suggestion
const include_path = "/include/";
```

keyword is awfully confusing